optimize/backend/src/main/java/io/camunda/optimize/service/db/repository/es/TaskRepositoryES.java

```diff
@@ -130,6 +135,25 @@ public boolean tryDeleteByQueryRequest(
                     .waitForCompletion(false)
                     .conflicts(Conflicts.Proceed));
 
+    if (configuration.getConnection().isHealthCheckEnabled()) {
```

Comment:
hmm I am wondering if we need to have a more generic parameter name here. This feels like for Optimize the health check property name is a bit misleading as we are also using it to determine other logic like this. Wdyt?

Reply:
fixed